src/Frontend/Themes/Bootstrap4/src/Layout/Templates/UserTemplates/test.html

```diff
@@ -3,4 +3,4 @@ <h2 data-ft-label="Title voor dit blokje" data-ft-type="text">Dit is een titel</
 <a href="#" data-ft-label="image link" data-ft-type="link-without-content"><img class="media-object" src="https://placehold.it/200x200" alt="placeholder" data-ft-label="Image" data-ft-type="image" data-ft-block-optional="true"></a>
 <p data-ft-label="beschrijving" data-ft-type="textarea">Dit is de beschrijving</p>
 <div data-ft-label="editor" data-ft-type="editor">Dit is de editor</div>
-<div class="image-holder" data-src="https://placehold.it/200x200" data-ft-label="Background image" data-ft-type="image-background" data-ft-block-optional="true" style="background-image: url('https://placehold.it/200x200');"></div>
+<div class="image-holder" data-src="https://placehold.it/200x200" data-ft-label="Background image" data-ft-type="image-background" style="background-image: url('https://placehold.it/200x200');"></div>
```

Comment:
Zou data-ft-block-optional wel laten staan

Reply:
voor de background zit die functionaliteit er niet in, lijkt mij ook niet nodig. Daarom heb ik die weggehaald